src/js/components/Navigation/HeaderBar.jsx

```diff
@@ -583,16 +609,6 @@ class HeaderBar extends Component {
       appBarCname += ' page-header__cordova';
     }
 
-    // We want the SelectBallotModal to appear on the ballot page (without a keystroke) if the page is empty and we have a textForMapSearch
```

Comment:
Hi @SailingSteve this fix was the result of a bug we found. Is there a reason to remove this?

Reply:
Go to the production wevote.us, clear all cookies, refresh and go to the ballot.  If you accept the default address, you go down a weird path, that doesn't take you to "seeing the ballot".   I suspected that was a bug.  Now you get the ballot select modal instead, and once you pick an election, you can see a ballot.

Reply:
Thank you for the explanation and the fix.